scripts/tgames.lic

```diff
@@ -333,7 +335,7 @@ def lootcheck(tier, game_location)
 			if result =~ /It is worth (.*) seashells./
 				found_seashells = $1.to_i
 				fput "redeem my seashell"
-				sleep 0.1
+				pause 0.25

```

Comment:
I'm curious why this change was necessary?  Wouldn't it be better to verify the result versus hoping the network roundtrip times match up?
```ruby
fput "redeem my seashell"
redeem_timeout = Time.now + 5 # seconds
wait_while { Char.right.noun.eql?("seashell") && Time.now < redeem_timeout }
fail "BOOM: tell Tysong something is wrong with redeeming" if Time.now > redeem_timeout
```

Reply:
About 1:25 times, the original 0.1 second pause would not be enough for the game-obj to update, verified that much with a couple people. It would probably be better to just look at the hands, and I could update it to be that instead.

Reply:
I think that would probably be best, at least if we are going to change the sleep, there is no guarantee even `0.25` is the correct duration to use, so it will invariably break for someone else again.

Reply:
maybe @mrhoribu wants to put some thoughts here, this is originally his script.